routes/assess.js

```diff
@@ -1,35 +1,19 @@
 const express = require('express');
-const firebase = require('firebase');
-const config = require('./config.js');
 const router = express.Router();
-
-//initializing firebase if its not already
-if(!firebase.apps.length) {
-   firebase.initializeApp(config.config);
-}
-
-// Setup database communication.
-const db = firebase.database();
-const ref = db.ref("live_weller");
-
-// Move to sub-directory.
-const marketsRef = ref.child("markets");
+const db = require('../db');
 
 router.get('/', function(req, res, next) {
-	let marketOptions = [];
+	db.getAllMarkets().then(function(result) {
```

Comment:
with the above change (returning JSON directly) this can become `.then(function(markets)`

Reply:
The .then attaches to the .once('value'), so I'm not sure if this works, I gave it a go returning the JSON inside a .then in the db section, but it didn't function this way.  See above for more detail.